py-polars/polars/series/datetime.py

```diff
@@ -1913,6 +1913,131 @@ def month_end(self) -> Series:
         ]
         """
 
+    def quarter_start(self) -> Series:
+        """
+        Roll backward to the first day of the quarter.
+
+        Returns
+        -------
+        Series
+            Series of data type :class:`Date` or :class:`Datetime`.
+
+        Notes
+        -----
+        If you're coming from pandas, you can think of this as a vectorised version
+        of `pandas.tseries.offsets.QuarterBegin().rollback(datetime)`.
```

Comment:
this should be
```
pandas.tseries.offsets.QuarterBegin(startingMonth=1).rollback(datetime)
```

(as an aside, `startingMonth` is a very odd argument name 😄 )